doc/stdlib/parallel.rst

```diff
@@ -355,18 +331,6 @@ General Parallel Computing Support
    * ``put!`` on a closed channel.
    * ``take!`` and ``fetch`` on an empty, closed channel.
 
-.. function:: RemoteRef()
```

Comment:
I made the changes in `base/docs/helpdb.jl`, built julia and then ran `./julia doc/genstdlib.jl`. I don't see the changes reflected in `doc/stdlib/parallel.rst`. Am I missing a step?